datasync/chaindatafetcher/kas/repository_transactions.go

```diff
@@ -128,7 +129,12 @@ func transformToTxs(event blockchain.ChainEvent) []*Tx {
 
 // InsertTransactions inserts transactions in the given chain event into KAS database.
 func (r *repository) InsertTransactions(event blockchain.ChainEvent) error {
-	return r.insertTransactions(transformToTxs(event))
+	txs := transformToTxs(event)
+	if err := r.insertTransactions(txs); err != nil {
+		logger.Error("Failed to insertTransactions", "err", err, "blockNumber", event.Block.NumberU64(), "numTxs", len(txs))
```

Comment:
`insertTransactions` already leaves logs internally.

Reply:
The internal logs were removed and the logs are pushed up.